src/ui/form/Button.tsx

```diff
@@ -1,31 +1,74 @@
 import { FC } from 'react';
 import styled from '@emotion/styled';
 
-const StyledButton = styled.button`
+const DefaultButton = styled.button`
   border: 1px var(--text) solid;
-  height: 30px;
   border-radius: 5px;
-  background: var(--input-background);
-  padding: 3px;
   &:disabled {
     opacity: 0.2;
   }
 `;
 
+const SmallButton = styled(DefaultButton)`
+  padding: 5px;
+`;
+
+const SalmonButton = styled(DefaultButton)`
+  background: var(--text);
+  color: var(--background);
+  padding: 10px;
+`;
+
+const WineButton = styled(DefaultButton)`
+  background: var(--input-background);
+  padding: 10px;
+`;
+const OchreButton = styled(DefaultButton)`
+  background: var(--ochre);
+  padding: 10px;
+  color: var(--background);
+  border-color: var(--background);
+  font-weight: 600;
+  font-size: 17px;
+  letter-spacing: 0.01em;
+  line-height: 122%;
```

Comment:
look up what people recommend about line-height. this shouldn't necessarily be something we're setting. i have a feeling you're using this like padding but maybe i'm wrong idk

Reply:
This is just straight from the Figma, I'll ditch it